atd-vzv/src/views/NotFound/UnderMaintenance.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import Footer from "../nav/Footer";
+
+import { Container, Row, Navbar, Alert } from "reactstrap";
+import { responsive } from "../../constants/responsive";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+
```

Comment:
you mind adding a comment block with a quick explainer of how to trigger this? it can be set via Netlify directory or by adding the `REACT_APP_UNDER_MAINTENANCE` env var to the start/build scripts.

Reply:
hmm, i guess maybe it'd be nice to go ahead and add this to the package.json start command, but as false? we could also add it to the build commands if we want to control via release/deploys...

```
"start": "REACT_APP_UNDER_MAINTENANCE=false REACT_APP_VZV_ENVIRONMENT=STAGING react-scripts start",
```